BUILD-CN.md

```diff
@@ -0,0 +1,45 @@
+
+# 构建 taosAdapter
+
+我们强烈建议将 taosAdapter 和 TDengine 服务端共同部署在同一个系统上，并使用官方 TDengine 安装包安装 taosAdapter。如果您想对 taosAdapter 进行调试或贡献代码，您也可以单独构建它。
+
+## 设置 golang 开发环境
+
+taosAdapter 是由 Go 语言开发的。关于 golang 开发环境的设置，请参考 golang 的[官方文档](https://go.dev/learn/)。
+
+请使用1.14以上版本的 golang。对于中国的用户，我们建议使用代理服务来加速软件包的下载。
+```
+go env -w GO111MODULE=on
+go env -w GOPROXY=https://goproxy.cn,direct
+```
+
+## 作为 TDengine 的一个组件构建 taosAdapter
+taosAdapter 的源代码是作为一个独立的代码库托管的，也通过子模块的方式存在于 TDengine 中。您可以下载 TDengine 的源代码并同时构建它们。步骤如下：
```

Comment:
Better to use consistant section style, place empty line between header and content.

Reply:
OK